lib/route/br_vlan.c

```diff
@@ -0,0 +1,1027 @@
+/* SPDX-License-Identifier: LGPL-2.1-only */
+/*
+ * lib/route/br_vlan.c		Bridge VLAN
+ */
+
+/**
+ * @ingroup rtnl
+ * @defgroup br_vlan Bridge VLAN
+ * @brief
+ * @{
+ */
+
+#include "netlink/attr.h"
+#include "netlink/msg.h"
+#include "nl-aux-route/nl-route.h"
+#include "nl-default.h"
+
+#include "nl-priv-dynamic-core/cache-api.h"
+#include <linux/if_bridge.h>
+
+#include <netlink/netlink.h>
+#include <netlink/route/br_vlan.h>
+#include <netlink/utils.h>
+#include <netlink/route/rtnl.h>
+#include <netlink/errno.h>
+#include <netlink/list.h>
+
+#include <stdint.h>
+#include <sys/socket.h>
+
+#include "nl-route.h"
+#include "nl-priv-dynamic-core/nl-core.h"
+#include "nl-priv-dynamic-core/object-api.h"
```

Comment:
includes should be ordered in a certain way. For example, `nl-default.h` must be first, because that one includes `config.h` (which must be first in autotools projects).

Something like:

```C
#include "nl-default.h"

#include <linux/if_bridge.h>

#include "netlink/attr.h"
#include "netlink/msg.h"
#include <netlink/netlink.h>
#include <netlink/route/br_vlan.h>
#include <netlink/utils.h>
#include <netlink/route/rtnl.h>
#include <netlink/errno.h>
#include <netlink/list.h>

#include "nl-route.h"
#include "nl-aux-route/nl-route.h"
#include "nl-priv-dynamic-core/nl-core.h"
#include "nl-priv-dynamic-core/cache-api.h"
#include "nl-priv-dynamic-core/object-api.h"
```